### DISCUSSION TITLE
bfe如何配置 Let's Encrypt 颁发的ssl证书

### DISCUSSION THREAD

**Discussion by hexergogo** [bfenetworks/bfe#1141] [Q&A (问题和回答)]
之前用的nginx 默认支持， https://certbot.eff.org/instructions?ws=nginx&os=ubuntufocal 按照教程按照下即可。
![image](https://github.com/bfenetworks/bfe/assets/46989742/afedbe0c-9165-4f14-b598-95457766c165)

到了bfe， 我能获取到Let's Encrypt 2个pem（key cert）证书， 但是不知道如何进行bfe的配置， 因为我看分为4个配置
<img width="1030" alt="image" src="https://github.com/bfenetworks/bfe/assets/46989742/6622bf5c-c4ec-4bab-841c-9a56465ecce8">

**Comment by mileszhang2016** [comment]
请参考
https://github.com/baidu/bfe-book/blob/version1/operation/configuration/https.md
中的说明
